### DISCUSSION TITLE
如何在vant中使用google icon

### DISCUSSION THREAD

**Discussion by tappmarket** [youzan/vant#10555] [Q&A] [answered]
https://fonts.google.com/icons

google icon有2,500多个不同的icon，而且可以调节weight和size。开发简单的app足够使用

在vant文档中只看到iconfront的使用方法，其它的icon该如何使用？

**Comment by chenjiahan** [comment, marked as answer]
参考 https://youzan.github.io/vant/#/zh-CN/icon#zi-ding-yi-tu-biao
